pages/api/og.tsx

```diff
@@ -43,22 +43,16 @@ export const config = {
 export default function (req: NextRequest) {
   try {
     const { searchParams } = new URL(req.url);
-    const testUrl = new URL(req.url);
-    const hasTitle = searchParams.has('title');
-    const title = hasTitle
-      ? searchParams.get('title')?.slice(0, 100)
-      : 'My Store';
-    const hasColor = searchParams.get('color')!;
-    const color = hasColor ? hasColor : '000000';
-    const foreground =
-      lightOrDark(`#${color}`) == 'light' ? '000000' : 'FFFFFF';
-    const logo = searchParams.get('logo');
+    const title = searchParams.get('title')?.slice(0, 100) || 'My Store';
```

Comment:
Is this backwards compatible?  If you pull a current image url and pass those params to a build of this do we get a reasonable image?  This change will need to go live before we can commit the storefront change.